### ISSUE TITLE
Does it have to be built with 5.4.0f3?

### ISSUE THREAD

**Issue by rodriguv** [KennyWdev/PokeLens-Demo#1]
When I go to build settings following the instructions from the academy link  (https://developer.microsoft.com/en-us/windows/holographic/holograms_100), I'm unable to list "microsoft hololens" as a device in the player settings. Are there any specifications to build it, like unity 5.4.0f3? There's no directions in the read me that shows how you built it so just wanted to see if there's anything i'm missing. I followed step-by-step from that link i posted.

**Comment by ToddBonnewell** [comment]
I ran it on 5.5. I think you are missing the Windows Store additional install from unity.

**Comment by ToddBonnewell** [comment]
If you have installed the Windows Store module, then you need to make sure you choose the Windows Store icon on the right tab. If you are in the other tab, you will not see it.

**Comment by rodriguv** [comment]
Thanks @ToddBonnewell ! The problem ended up being that I had to download unity 5.5.0f3 with the following components at the beginning of the download assistant:
- Windows Store Playback Engine (.NET Scripting Backend)
- Windows Store Playback engine (IL2CPP Scripting Backend)
- Microsoft Visual Studio Tools for Unity

I had included the Microsoft visual studio tools with the previous Unity installation, but not the two other scripting backends.

**Comment by ToddBonnewell** [comment]
Glad to hear! You could have done the same with the previous version of Unity. But it is always good to get the latest bits.

**Comment by vroomcam** [comment]
Yes, the updates do get challenging and requiring uninstalling and reinstalling some sdk's, sometimes even resetting and restarting. I notices some Windows updates also caused some impediments, thank you for updates and details.
